UPGRADE-6.3.md

```diff
@@ -6,6 +6,11 @@ DependencyInjection
 
  * Deprecate `PhpDumper` options `inline_factories_parameter` and `inline_class_loader_parameter`, use `inline_factories` and `inline_class_loader` instead
 
+FrameworkBundle
+---------------
+
+ * Deprecate `framework:exceptions` tag, unwrap it and replace `framework:exception` tags’ `name` attribute by `class`.
```

Comment:
Can we add a before/after example?

Reply:
Yes! https://github.com/symfony/symfony/pull/48665/commits/39be25d2e76453ca9b6f554e701cbfe4b7126fe3

Reply:
final dot should be removed + let's use ASCII: `s/’/'/`